src/main/groovy/com/rundeck/plugins/ansible/plugin/AnsibleResourceModelSource.java

```diff
@@ -812,4 +819,23 @@ public List<String> listSecretsPathResourceModel(Map<String, Object> configurati
 
   }
 
+  /**
+   *
+   * @param value parameter from gui
+   * @param paramName parameter name
+   * @return an integer value
+   * @throws ConfigurationException
+   */
+  private Integer getIntegerValue(String value, String paramName) throws ConfigurationException {
```

Comment:
this is node needed if the field plugin type is integer

Reply:
done... ✅